app/routes.py

```diff
@@ -1,2 +1,284 @@
-from flask import Blueprint
+from flask import Blueprint, request, jsonify, make_response
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+import requests
+from dotenv import load_dotenv
 
+load_dotenv()
+
+task_list_api_bp = Blueprint("task_list_api", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@task_list_api_bp.route("", methods=["GET", "POST"])
+def tasks():
```

Comment:
Because GET and POST are not using the same code you could consider created two different methods to handle GET and POST. For example, `get_tasks` and `create_task`